### PR TITLE
Porting opi_recognition to ROS2

### PR DESCRIPTION
**Pull request by IliTheButterfly**
# Created packages:
## rove_opi:
Takes in an image message and returns a polygon message. The polygon is in 2D (only the x and y values of the points are used).
Use `ros2 launch rove_opi opi.launch.py` to run.

## rove_opi_publisher:
Used for debugging.
Use `ros2 launch rove_opi test_opi.launch.py` to run

### CODE REVIEW

File: src/rove_opi/package.xml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>rove_opi</name>
+  <version>0.0.0</version>
+  <description>TODO: Package description</description>
+  <maintainer email="jocobc@todo.todo">jocobc</maintainer>
```
Comment (SimonR99) [inline src/rove_opi/package.xml:7]: change package maintainer with the capra email

File: src/rove_opi/rove_opi/ReadMe.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+On ubuntu it should just be:
+```bash
+sudo apt install tesseract-ocr
+```
+
+Then install packages:
+
+```bash
+python -m pip install pip --upgrade
+pip install -r src/rove_opi/rove_opi/requirements.txt
```
Comment (SimonR99) [inline src/rove_opi/rove_opi/ReadMe.md:17]: le dockerfile n'a pas été updaté, quand on va faire des test, les builds ne passeront probablement pas

**Review by SimonR99 — COMMENTED** [review]
We need to discuss this, I'm not sure rove_opi should be in the rove repo. I think it should be a standalone repo, something like capra_opi. Then we would have .repos that will be in charge of getting it. That way, other teams or even us later on a new robot would be able to reuse it. A simple launchfile in rove_bringup would call the package.

File: src/rove_opi/rove_opi/ReadMe.md (hunk 2 above)
Comment (IliTheButterfly) [reply src/rove_opi/rove_opi/ReadMe.md]: J'ai ajouté les packages dans le setup.py et j'ai mis le apt install de tesseract dans le dockerfile.

**Pull request closed without merging**